projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -67,6 +81,9 @@ export function isXyzOffering(str: string): str is Xyz_Offering {
 export function isGeoJsonOffering(str: string): str is GeoJson_Offering {
   return str === 'http://www.opengis.net/spec/owc-geojson/1.0/req/geojson';
 }
+export function isTMSOffering(str: string): str is TMS_Offering {
+  return str === 'https://wiki.osgeo.org/wiki/Tile_Map_Service_Specification';
```

Comment:
We should stay to the url format `http://www.opengis.net/spec/owc-geojson/1.0/req/...` defined in the OGC OWS Context https://docs.opengeospatial.org/is/14-055r2/14-055r2.html#97. I think it's just a dummy url anyway.

Reply:
Has been updated!